### PR TITLE
feat(db):Completed updateCart and getOrderById functions

### LINKED ISSUE #30: db methods for Cart

### PR DESCRIPTION
**Pull request by jhoolwerf**

### CODE REVIEW

**Review by KayPaint — APPROVED** [review]

File: db/models/cart.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
+const client = require("./client");
+
+async function addItemToCart({ orderId, quantity }) {
+    try {
+        const { rows: [newOrder] } = await client.query(`
+        INSERT INTO cart("orderId", quantity)
```
Comment (Remics89) [inline db/models/cart.js:6]: we can add orderId and quantity but the cart table definition has to reflect that. CREATE TABLE cart should have orderId and quantity defined. Ive created an issue based on this. #45

File: db/models/cart.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    } catch (error) {
+        console.error("Could not get order by id");
+        throw error;
+    }
+}
+
+async function updateCart({ id }) {
+    try {
+        const order = await getOrderById(id);
+        const fields = {};
```
Comment (Remics89) [inline db/models/cart.js]: fields here is undefined. Then at line 40, fields is called to create the setString. In this scenario, setString will always be undefined and hit your return at 44.

File: db/models/cart.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
+const client = require("./client");
```
Comment (Remics89) [inline db/models/cart.js]: - require client statement should have 2 dots to reference a file outside the immediate directory.

**Review by Remics89 — CHANGES_REQUESTED** [review]

File: db/models/cart.js (hunk 3 above)
Comment (Remics89) [reply db/models/cart.js]:
```suggestion
const client = require("../client");
```

File: db/models/cart.js (hunk 2 above)
Comment (Remics89) [reply db/models/cart.js:36]: line 33 function arguments:
```suggestion
        {id, ...fields}
```

**Review by Remics89 — APPROVED** [review]

**Pull request merged**